src/bundles/files/actions.js

```diff
@@ -620,7 +652,7 @@ const dirStats = async (ipfs, cid, { path, isRoot, sorting }) => {
     const absPath = join(path, f.name)
     let file = null
 
-    if (dirCount < 1000 && (f.type === 'directory' || f.type === 'dir')) {
```

Comment:
why we change this?

https://github.com/ipfs/ipfs-webui/blob/d234a6012a899e52fdac1c44730c0e7f7af6412e/src/bundles/files/actions.js#L47 still refers to `directory` 


avoid touching things unrelated to CAR import :pray: